src/paths.ts

```diff
@@ -225,6 +225,29 @@ class Paths {
         }
     }
 
+    async getCPackPath(wsc: DirectoryContext, overWriteCMakePathSetting?: string): Promise<string | null> {
+        const cpackPath = await this.expandStringPath(wsc.config.rawCPackPath, wsc);
+        if (!cpackPath || cpackPath === 'auto' || overWriteCMakePathSetting) {
```

Comment:
Is this because we do this in other locations in the codebase? What does the `auto` string actually do for us? I've not noticed this before, so I may just be ignorant to this feature.